bin/goto

```diff
@@ -37,37 +37,6 @@ function _goto_main {
         fi
 
 
-        # Special case 1 (goto cd <magicword>)
-        if [ "$1" = "cd" ]; then
```

Comment:
Need help understanding why special case 1 and 2 blocks `stdin` from passing to `the_real_goto`

Reply:
Not sure, will conduct tests. I believe we would save us self of lots of complexity if we add the explicit mark of taking input from stdin: `-`

I don't have my copy of the UNIX Programming Environment at hand right now, but I recall there was a mention of this convention in there :)